pkgs/development/python-modules/ratelimiter/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  stdenv
+, buildPythonPackage
+, fetchPypi
+, pytest
+}:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "ratelimiter";
+  version = "1.2.0.post0";
+
+  propagatedBuildInputs = [];
+
+  src = fetchPypi {
+      inherit pname version;
+      sha256 = "5c395dcabdbbde2e5178ef3f89b568a3066454a6ddc223b76473dac22f89b4f7";
+  };
+
+  prePatch = ''
```

Comment:
Unfortunately, Python < 3.7 requires `glibcLocales` in `nativeBuildInputs` and something like `LC_ALL = "en_US.UTF-8";` in the expression. But it is still better than `sed`ing.